blog/2016-12-05-easy-upgrades-processed-by-git.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Easier Upgrades
+author: Nicolas Cuillery
+authorTitle: JavaScript consultant and trainer at Zenika
+authorURL: https://twitter.com/ncuillery
+authorImage: https://fr.gravatar.com/userimage/78328995/184460def705a160fd8edadc04f60eaf.jpg?size=128
+authorTwitter: ncuillery
+category: announcements
+---
+
+Upgrading to each new version of React Native has been painful. You might have seen something like this before:
```

Comment:
I've noted that the first sentence is actually the teaser seen on the [home page](http://facebook.github.io/react-native/blog/). That's why I went with a long sentence introducing the blog post and encouraging people to read it:

>Each release of React Native may come with changes inside the iOS and Android sub-projects that you have to report in your project. This synchronisation has always been a major pain point because of the poor changes detection within your source files. Today, I'm proud to announce a new upgrading process relying on the most popular version control system: Git. 

At the current time, the home page looks like this: 
<img width="640" alt="capture d ecran 2016-12-03 a 13 53 46" src="https://cloud.githubusercontent.com/assets/4425455/20859483/f9686658-b960-11e6-9f66-30a58977edd9.png">